src/lib/profiles/software-update/SoftwareUpdateProfile.cpp

```diff
@@ -28,460 +28,575 @@
 #include <Weave/Profiles/ProfileCommon.h>
 #include "SoftwareUpdateProfile.h"
 using namespace ::nl::Weave;
-using namespace ::nl::Weave::Profiles;
-using namespace ::nl::Weave::Profiles::SoftwareUpdate;
```

Comment:
Allowing the compiler derive full class names from the `using` statement seemed unnecessary and sloppy.  Moved all the code within a proper namespace definition

Reply:
+1